RecoLocalTracker/SiPixelRecHits/plugins/PixelCPEClusterRepairESProducer.cc

```diff
@@ -34,16 +34,17 @@ class PixelCPEClusterRepairESProducer : public edm::ESProducer {
   edm::ESGetToken<SiPixel2DTemplateDBObject, SiPixel2DTemplateDBObjectESProducerRcd> templateDBobject2DToken_;
 
   edm::ParameterSet pset_;
-  bool DoLorentz_;
+  bool doLorentzFromAlignment_;
+  bool useLAFromDB_;
 };
 
 using namespace edm;
 
 PixelCPEClusterRepairESProducer::PixelCPEClusterRepairESProducer(const edm::ParameterSet& p) {
   std::string myname = p.getParameter<std::string>("ComponentName");
 
-  //DoLorentz_ = p.getParameter<bool>("DoLorentz"); // True when LA from alignment is used
-  DoLorentz_ = p.getParameter<bool>("DoLorentz");
+  useLAFromDB_ = p.getParameter<bool>("useLAFromDB");
+  doLorentzFromAlignment_ = p.getParameter<bool>("doLorentzFromAlignment");
```

Comment:
there is no matching update in the `::fillDescriptions` method.
There is a comment that `DoLorentz` was specific to this ES producer; since it's no longer needed, please remove it or mark it as obsolete

Reply:
Ok I removed the the line from the fillDescriptions method